howto/how-to-set-the-policy-for-the-transaction-fee.md

```diff
@@ -0,0 +1,394 @@
+---
+title: "How to set the policy for the transaction fees"
+excerpt: ""
+---
+
+## Overview
+
+Users must pay the transaction fees to execute a smart contract (SCORE) on the ICON Network. But the fact that all DApp users were required to pay transaction fees has been a major hurdle to attract more users into the DApp service. With the Fee 2.0 feature (Fee Sharing and Virtual Step), the DApp service operators will be able to pay the transaction fees on behalf of the service users.
```

Comment:
I think it would be better to use the terminology 'DApp' or 'dApp' in a consistent way. In this comment 'DApp' changed into 'dApp' but I found that the 'DApp' is used on following description.(Line 14 of this docs)